rocky/tools/models.py

```diff
@@ -106,6 +106,9 @@ class Meta:
             ("can_enable_disable_boefje", "Can enable or disable boefje"),
             ("can_set_clearance_level", "Can set clearance level"),
             ("can_delete_oois", "Can delete oois"),
+            ("can_mute_findings", "Can mute findings"),
+            ("can_view_katalogus_settings", "Can view KAT-alogus settings"),
+            ("can_set_katalogus_settings", "Can set KAT-alogus settings"),
```

Comment:
I believe we do not check if someone who has `can_view_katalogus_settings` but not `can_set_katalogus_settings` can see but not set Settings. We only test having both or neither.

Reply:
Yeah in that case I also wouldn't fix that at this point